firmware/laseramp.ino

```diff
@@ -1,143 +1,251 @@
+/*
+Release 0.1
+We ignore N commands.
+
+Todo:
+Support for N\d+ line prefix.
+Set on run time the size of bed, the steps per mm
+Save configuration to eeprom
+Save to eeprom default values.
+Finish Raster mode.
+Support for negative values in X, Y, Z.
+Add Homing option for HARDWARE positioning and WORK positioning
+
+http://www.deskcnc.com/Homing.html
+Raster Mode
+G5 <[0-9A-F]>
+each char is a 8bit long code, each command may send up to 64 bytes.cada
```

Comment:
true that hexa is 4 bits, but in order to send those values I will use chars therefore the 8-bit. I do not want to send non printable characters as it make it difficult to manually send or edit.